redash/handlers/users.py

```diff
@@ -40,6 +40,18 @@ def invite_user(org, inviter, user):
     send_invite_email(inviter, user, invite_url, org)
 
 
+def bool_or_none(v):
```

Comment:
![image](https://user-images.githubusercontent.com/289488/52305664-59392680-299e-11e9-9135-587566df3a8a.png)

Reply:
Okay, I'll move this to handler

Reply:
If we show them on a different tab (makes sense), how about using a dedicated API call (`/api/users/invites`), so we don't need this at all?

Reply:
I don't think it's a good idea - implementation of a new endpoint handler will be 99% the same (with search support, pagination, etc.)

Reply:
@arikfr (cc @rauchy) What is I'll add `get` handler to [UserInviteResource](https://github.com/getredash/redash/blob/master/redash/handlers/users.py#L132)?

Reply:
Nevermind, I'll create a new resource for this